src/Makefile.am

```diff
@@ -121,6 +121,7 @@ AM_LDFLAGS =  $(LDFLAGS)
 AM_LDFLAGS += $(VMI_LIBS)
 AM_LDFLAGS += $(GLIB_LIBS)
 AM_LDFLAGS += $(JSONC_LIBS)
+AM_LDFLAGS += -lz
```

Comment:
You shouldn't have to add flags manually

Reply:
Fixed